.eslintrc.js

```diff
@@ -110,44 +119,19 @@ module.exports = {
       'extends': ['plugin:jest/recommended', 'plugin:jest/style'],
     },
     {
-      'files': ['test/**/*.test.js'],
+      'files': ['test/**/*.test.js', 'test/**/*.test.mjs'],
       'rules': {
+        'import/namespace': 'off',
         'jest/expect-expect': 'off',
         'jest/no-conditional-expect': 'off',
         'jest/no-done-callback': 'off',
-        'jest/no-standalone-expect': 'off',
-        'jest/no-try-expect': 'off',
```

Comment:
Are these rules not enforced anymore?

Reply:
That's other way around. These rules been off for some reason, but it is not necessary as linter passes without those overrides, so here we are enforcing them now.